scripts/geo.py

```diff
@@ -57,31 +56,60 @@
                 geo.py cofa              will provide the cofa information
                 geo.py cofa --graph(-g)  will plot cofa with "background" set in --station-types
 
-            corr:  Provides information on the correlator hookups.
-                geo.py corr:  will provide a list of correlator inputs for stations in --station-types
-                geo.py corr --loc(-l)              will provide the correlator input for --loc
-                geo.py corr --loc(-l) --graph(-g)  will provide/plot correlator info for --loc
-
             since: Provides a list of all antennas installed --since DATE
                  geo.py since --date DATE             will provide a list of all antennas built --since DATE
                  geo.py since --date DATE --graph(-g)  will provide a list/plot those antennas
 
             info:  Print this information and exit.
                 geo.py info
+
+        usage: geo.py [-h]
```

Comment:
Is this just a giant copy-paste of the help text producted by the `argparse` module? Surely there's a better way to emit that information.

Reply:
Well, it is a slightly edited one for clarity -- I wanted `info` to be one-shop stop.

Reply:
It would be nice if this help text could leverage the argparser's knowledge more, since I predict that they will get out of sync in hard-to-track ways. But that's a bridge we can cross later.